### PR TITLE
chore: [IOBP-111,IOBP-116] Addition of LogoPaymentHuge and usage in new Cards components

### PR DESCRIPTION
**Pull request by forrest57**
DEPENDS ON https://github.com/pagopa/io-app/pull/4759
## Short description
|Screenshot 2023-07-13 at 15 36 26|Screenshot 2023-07-13 at 15 59 55|

Added new LogoPaymentHuge component and applied it to PaymentCardBig

## List of changes proposed in this pull request
- added LogoPaymentExtended component
- refactored folder structure for the cards section
- added BankLogoOrSkeleton component, which takes an ABI code and displays a loading skeleton until the resource is loaded
- refactored some points of PaymentCardBig
- added new utility function which basically is a template string for the ABI CDN URL
- added the new components to DSLogos

## How to test
run the tests, also navigate to the Design System  (Logos section and Cards section) and make sure the components are rendered as shown in the image and as shown in the design.

### CODE REVIEW

File: ts/components/ui/LogoPaymentHuge.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+} & (
+  | {
+      icon: "payPal" | "bpay";
+    }
+  | {
+      abiCode: string;
+    }
+);
+
+export const LogoPaymentHuge = (props: LogoPaymentHugeProps) => {
```
Comment (mastro993) [inline ts/components/ui/LogoPaymentHuge.tsx:18]:
```suggestion
export const LogoPaymentExtended = (props: LogoPaymentHugeProps) => {
```
Sounds like a more appropriate name. What do you think? `Huge` refers to the size, but here we have an **extended** version of the logo, which can be the same size or even smaller than the standard one.

**Review by mastro993 — CHANGES_REQUESTED** [review]

File: ts/components/ui/LogoPaymentHuge.tsx (hunk 1 above)
Comment (forrest57) [reply ts/components/ui/LogoPaymentHuge.tsx]: addressed in [92989bc](https://github.com/pagopa/io-app/pull/4781/commits/92989bcc9bd38990fbb30e41b1b7d55406f2f8f2)

**Review by mastro993 — CHANGES_REQUESTED** [review]

File: ts/components/ui/utils/helperFunctions/payment/strings.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+/**
```
[the remark this answers is not in the document]
Comment (forrest57) [reply ts/components/ui/utils/helperFunctions/payment/strings.ts]: adressed in [bfc816a](https://github.com/pagopa/io-app/pull/4781/commits/bfc816a96199a895f3036faf1fbdbbd56bae72a0)

File: ts/components/ui/utils/helperFunctions/generateFlatListItemLayout.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
+type ItemLayout = {
```
[the remark this answers is not in the document]
Comment (forrest57) [reply ts/components/ui/utils/helperFunctions/generateFlatListItemLayout.ts]: adressed in [bfc816a](https://github.com/pagopa/io-app/pull/4781/commits/bfc816a96199a895f3036faf1fbdbbd56bae72a0)

**Pull request merged**